### PR TITLE
Add non-confirmation-bias mode

### PR DESCRIPTION
**Pull request by Myzel394**
People believe some people more than others. We believe for example that Albert Einstein knows more about physics more than our grandmother, just because it's Albert Einstein. Only a few people would actually check his work and see if he was right or not.

To avoid this confirmation bias based on authority, you should have an option to hide the authors, so that you don't know who made a tweet.

This is what this PR accomplishes.

Please let me know how you like it and whether we need to add something more to add / change something.

Btw, @jonjomckay that's a very clean codebase! It was very easy to create this PR, thanks for your awesome work! :heart: :)

### CODE REVIEW

File: lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -74,6 +74,12 @@ class MessageLookup extends MessageLookupByLibrary {
         "about": MessageLookupByLibrary.simpleMessage("About"),
         "account_suspended":
             MessageLookupByLibrary.simpleMessage("Account suspended"),
+        "activate_non_confirmation_bias_mode_description":
+            MessageLookupByLibrary.simpleMessage(
+                "Don\'t show the author of the tweet on a post. Avoid confirmation bias based on authoritative arguments."),
```
Comment (jonjomckay) [inline lib/generated/intl/messages_en.dart:79]: I think this description is a bit long, and probably won't fit on one line on most devices. Maybe truncate it to `Don't show tweet authors`, as the title declares it as "non-confirmation bias mode" already

File: lib/generated/intl/messages_en.dart [hunk 2]
```diff
@@ -74,6 +74,12 @@ class MessageLookup extends MessageLookupByLibrary {
...
+        "activate_non_confirmation_bias_mode_label":
+            MessageLookupByLibrary.simpleMessage(
+                "Activate non-confirmation-bias-mode"),
```
Comment (jonjomckay) [inline lib/generated/intl/messages_en.dart:82]: I think this should just be `Activate non-confirmation bias mode`, as this is an open compound

File: lib/main.dart [hunk 3]
```diff
@@ -180,6 +177,7 @@ Future<void> main() async {
     optionThemeTrueBlack: false,
     optionThemeColorScheme: 'aquaBlue',
     optionTrendsLocation: jsonEncode({'name': 'Worldwide', 'woeid': 1}),
+    optionImproveNonConfirmationBias: false,
```
Comment (jonjomckay) [inline lib/main.dart:180]: `optionNonConfirmationBiasMode` fits better, as it's more 1:1 with the name of the preference.

**Review by jonjomckay — CHANGES_REQUESTED** [review]
Thanks for this! I've added a few comments to address before I can merge it in. There's also a new conflict, as I've pushed since the last commit (sorry).

P.S. for future PRs, it's helpful to exclude formatting changes that are unrelated to the change that the PR targets - it makes it a lot easier to review. `git add -p` is helpful here 🙂

**Comment by Myzel394** [comment]
@jonjomckay Should be fixed everything. Before merging you may look if you like this design change, I think we should add a button like "Go to author" or "Show author" so that people know this area is clickable.

**Comment by Myzel394** [comment]
btw, shouldn't `lib/generated` be git-excluded?

File: lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (jonjomckay) [reply lib/generated/intl/messages_en.dart]: This one's still outstanding, I think

**Comment by jonjomckay** [comment]
> btw, shouldn't `lib/generated` be git-excluded?

Probably yeah. I'll look into it!

File: lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (Myzel394) [reply lib/generated/intl/messages_en.dart]: Take a look at the `intl_en.arb` file, the `messages_en.dart` is generated based on it.

**Comment by Myzel394** [comment]
While testing out the beta version, I realized that the current implementation feels a bit off. It surely does because you are used to see an user icon and a name - I thought about how we add a generic user profile and the name "user", so it doesn't feel that off. What do you think @jonjomckay? I can create a PR if you'd like

**Comment by jonjomckay** [comment]
@Myzel394 that sounds good to me! It does feel a little strange seeing a black box there 😅

**Pull request merged**